MetaMask/metamask-mobile
# feat: minor changes to connect QR wallet screen

## **Description**



This PR introduces minor UI improvements to the Connect QR wallet screen

## **Related issues**

Fixes: https://github.com/MetaMask/metamask-mobile/issues/7002

## **Manual testing steps**

1. Go to "Add account or hardware wallet" > "Add hardware wallet" > "QR wallet"
2. Verify the UI changes

## **Screenshots/Recordings**



### **Before**

<img src="https://github.com/MetaMask/metamask-mobile/assets/17601467/b40b1a86-3c22-4395-a238-ebac55a686aa" width="250" /> <img src="https://github.com/MetaMask/metamask-mobile/assets/17601467/650cf65c-c0d5-40ab-86b5-df8d6b277f5d" width="250" />

### **After**

<img src="https://github.com/MetaMask/metamask-mobile/assets/17601467/839e095e-7bce-47af-8ed4-4977067360f3" width="250" /> <img src="https://github.com/MetaMask/metamask-mobile/assets/17601467/e3551580-896f-4aa4-ae78-4930f4e9e5d7" width="250" />




## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
